packages/persistent-merkle-tree/src/node.ts

```diff
@@ -227,9 +227,11 @@ export class LeafNode extends Node {
       let h = getNodeH(this, hIndex);
       if (uintBytes === 1) {
         h &= ~(0xff << bitIndex);
+        // @ts-expect-error - This kind of expression is always truthy - Identified after TS 5.8 upgrade.
         h |= (0xff && value) << bitIndex;
```

Comment:
Ah, I suppose its found a bug. I think this is the intended
```suggestion
        h |= (0xff & value) << bitIndex;
```